resources/sdk/purecloudjava/templates/README.mustache

```diff
@@ -133,13 +133,13 @@ Specify the connector in the builder:
 By default, the Java SDK does not automatically retry any failed requests.
 To enable automatic retries, provide a RetryConfiguration object with the maximum number of seconds to retry requests when building the ApiClient instance.
 
-Building a `RetryConfiguration` instance:
+Building a `RetryConfiguration` instance:  
 ```{"language":"java"}
 ApiClient.RetryConfiguration retryConfiguration = new ApiClient.RetryConfiguration();
 retryConfiguration.setMaxRetryTimeSec(30);
 ```
 
-Setting `RetryConfiguration` instance to `ApiClient`:
+Setting `RetryConfiguration` instance to `ApiClient`:  
  ```{"language":"java"}
```

Comment:
There's a space at the beginning of this line that's causing the code fence not to render properly.